docs/csharp/asynchronous-programming/index.md

```diff
@@ -202,6 +202,9 @@ When code running asynchronously throws an exception, that exception is stored i
 
 The most common scenario for a faulted task is that the `Exception` property contains exactly one exception. When code `awaits` a faulted task, the first exception in the <xref:System.AggregateException.InnerExceptions?displayProperty=nameWithType> collection is rethrown. That's why the output from this example shows an `InvalidOperationException` instead of an `AggregateException`. Extracting the first inner exception makes working with asynchronous methods as similar as possible to working with their synchronous counterparts. You can examine the `Exception` property in your code when your scenario may generate multiple exceptions.
 
+> [!TIP]
+> We recommend that any argument validation exceptions emerge *synchronously* from task-returning methods. For more information and an example of how to do it, see [Exceptions in task-returning methods](../fundamentals/exceptions/creating-and-throwing-exceptions.md#exceptions-in-task-returning-methods).
```

Comment:
```suggestion
> It's recommended that any argument validation exceptions emerge *synchronously* from task-returning methods. For more information and an example of how to do it, see [Exceptions in task-returning methods](../fundamentals/exceptions/creating-and-throwing-exceptions.md#exceptions-in-task-returning-methods).
```

Reply:
I thought this ("we recommend") was a case where they relaxed the rules about use of "we". Let me see if I can dig that up.

Reply:
https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=36430